### ISSUE TITLE
Disk/Volume usage growing without any clear reason.

### ISSUE THREAD

**Issue by Gradlon** [seaweedfs/seaweedfs#5823]
**Describe the bug**
I have used seaweedfs for a few weeks.
I started to notice that used storage and volume count keeps increasing even though nothing is added.
The storage is taken by the default collection, so it is easy to delete whatever the content of it is.
The only files that I see being added are the logs, but the size does not fit.

I expect some overhead but not that much, basically the disk are filling up within a week.

**System Setup**
Base os: RockyLinux 9
Docker: 26
```
services:
  volume:
    image: chrislusf/seaweedfs:3.69_large_disk_full
    container_name: seaweedfs-volume
    network_mode: "host"
    command: >
      -v=2
      volume 
      -dataCenter=${DATACENTER} 
      -rack=${RACK} 
      -mserver=10.0.0.15:9333,10.0.0.11:9333,10.0.0.13:9333
      -ip=${IP} 
      -ip.bind=0.0.0.0 
      -port=9090  
      -metricsPort=9325
      -dir=/data/volumes/01,/data/volumes/02
      -max=0,0
      -idleTimeout=60
      -index=leveldbMedium
      -hasSlowRead=true
      -readBufferSizeMB=64
      -concurrentDownloadLimitMB=10000
      –concurrentUploadLimitMB=10000
      -fileSizeLimitMB=4096
    volumes:
      - /data/bin/curl:/bin/curl:z,ro
      - /data/seaweedfs/volumes:/data/volumes:Z
      - ./config:/etc/seaweedfs:Z,ro
    ulimits:
      memlock:
        soft: -1
        hard: -1
      nofile:
        soft: 65536
        hard: 65536

services:
  master:
    image: chrislusf/seaweedfs:3.69_large_disk_full
    container_name: seaweedfs-master
    network_mode: "host"
    command: >
      -v=2
      master
      -peers=${PEERS}
      -defaultReplication=010
      -volumeSizeLimitMB=1024
      -garbageThreshold=0.1
      -volumePreallocate
      -ip=${IP}
      -raftHashicorp
      -ip.bind=${BIND}
      -metricsPort=9324
    volumes:
      - /data/bin/curl:/bin/curl:z,ro
      - ./config:/etc/seaweedfs:Z,ro
      - /data/seaweedfs/master:/data:Z
```

**Expected behavior**
Storage to remain stable if nothing is added
**Screenshots**
<img width="1725" alt="image" src="https://github.com/user-attachments/assets/dd7f6f4d-e95b-49f4-bfe4-a726619a9238">

**Additional context**
Three master nodes
two storage nodes

**Comment by chrislusf** [comment]
It's strange if nothing is added.

**Comment by Gradlon** [comment]
Yes indeed.
Actually, if I add file, it grows quicker.
![image](https://github.com/user-attachments/assets/d637c823-7d6a-4f65-841f-dd27ef0418d7)

This is the master config
```
# weed scaffold -config=master
# Put this file to one of the location, with descending priority
#    ./master.toml
#    $HOME/.seaweedfs/master.toml
#    /etc/seaweedfs/master.toml
# this file is read by master

[master.maintenance]
# periodically run these scripts are the same as running them from 'weed shell'
scripts = """
  lock
  volume.deleteEmpty -quietFor=24h -force
  volume.balance -force
  volume.fix.replication
  s3.clean.uploads -timeAgo=24h
  unlock
"""
sleep_minutes = 17          # sleep minutes between each script execution


[master.sequencer]
type = "raft"     # Choose [raft|snowflake] type for storing the file id sequence
# when sequencer.type = snowflake, the snowflake id must be different from other masters
sequencer_snowflake_id = 0     # any number between 1~1023


# configurations for tiered cloud storage
# old volumes are transparently moved to cloud for cost efficiency
[storage.backend]
[storage.backend.s3.default]
enabled = false
aws_access_key_id = ""         # if empty, loads from the shared credentials file (~/.aws/credentials).
aws_secret_access_key = ""     # if empty, loads from the shared credentials file (~/.aws/credentials).
region = "us-east-2"
bucket = "your_bucket_name"    # an existing bucket
endpoint = ""
storage_class = "STANDARD_IA"

# create this number of logical volumes if no more writable volumes
# count_x means how many copies of data.
# e.g.:
#   000 has only one copy, copy_1
#   010 and 001 has two copies, copy_2
#   011 has only 3 copies, copy_3
[master.volume_growth]
copy_1 = 3                # create 1 x 7 = 7 actual volumes
copy_2 = 6                # create 2 x 6 = 12 actual volumes
copy_3 = 3                # create 3 x 3 = 9 actual volumes
copy_other = 1            # create n x 1 = n actual volumes

# configuration flags for replication
[master.replication]
# any replication counts should be considered minimums. If you specify 010 and
# have 3 different racks, that's still considered writable. Writes will still
# try to replicate to all available volumes. You should only use this option
# if you are doing your own replication or periodic sync of volumes.
treat_replication_as_minimums = false
```

And this is the filer
```
  filer:
    image: chrislusf/seaweedfs:3.69_large_disk_full
    container_name: seaweedfs-filer
    labels:
      ch.integral-systems.group: "SeaWeedFS"
      ch.integral-systems.deployment: "filer"
      ch.integral-systems.health_monitor: "true"
      ch.integral-systems.customer: "false"
      ch.integral-systems.infrastructure: "true"
      ch.integral-systems.services: "false"
      ch.integral-systems.internal: "true"
    network_mode: "host"
    command: '-v=2 filer -master=10.0.0.15:9333,10.0.0.11:9333,10.0.0.13:9333 -ip.bind=0.0.0.0 -metricsPort=9326 -collection=system'
    volumes:
      - /data/bin/curl:/bin/curl:z,ro
      - /data/seaweedfs/filer:/data:Z
      - ./config:/etc/seaweedfs:Z,ro
    tty: true
    stdin_open: true
    healthcheck:
      test: [ "CMD", "curl", "--fail", "-I", "http://localhost:8888" ]
      interval: 1s
      start_period: 10s
      timeout: 30s
      ```

**Comment by chrislusf** [comment]
the disk usage growth is proportional to the updates, since all updates are append-only.

**Comment by Gradlon** [comment]
Yes, that would make sense, but the growth happens on the system collection but the files where added to another collection.

Do you have any idea where I could look to find more information, why this issue happnes?

**Comment by Gradlon** [comment]
I made some further investigations but could not find the reason for this behavior.
The only thing that I can try now is to switch to the 32GB version and see if that is the reason.
The default collection is growing, but there is A. Na data except for topics, and there is no one to connect to this collection. 
To prevent a complete fill up of the cluster I deleted the entire collection, which does not seem to have any effect on the cluster.

But of course, this is not a solution.
